dcc-portal-ui/app/scripts/advanced/js/advanced.js

```diff
@@ -53,9 +53,9 @@ angular.module('icgc.advanced', ['icgc.advanced.controllers', 'ui.router'])
 angular.module('icgc.advanced.controllers', [
     'icgc.advanced.services', 'icgc.sets.services', 'icgc.facets'])
     .controller('AdvancedCtrl',
-    function ($scope, $rootScope, $state, $modal, Page, AdvancedSearchTabs, LocationService, AdvancedDonorService,
+    function ($scope, $rootScope, $state, $modal, Page, AdvancedSearchTabs, LocationService, AdvancedDonorService, // jshint ignore:line
```

Comment:
Added jshint ignore here as our AdvancedCtrl has too many dependencies (17 > 16).  We should investigate if maybe this can be simplified.